### ISSUE TITLE
Gene-set burden tests

### ISSUE THREAD

**Issue by Wainberg** [rgcgithub/regenie#420]
Gene-set burden tests where the genes span multiple chromosomes are difficult to run because regenie [only supports running on a single genotype file set at a time](https://github.com/rgcgithub/regenie/issues/212#issuecomment-971645954). In the common case where there's one genotype file per chromosome, that means merging across chromosomes, which is painful for a dataset like the UK Biobank exomes. Would it be possible to revisit support for specifying multiple file sets to regenie at once?

**Comment by Wainberg** [comment]
Here's an example where the authors used regenie for their single-gene burden tests but not for their multi-gene burden tests, possibly because of this limitation: panel A of [this figure](https://www.nature.com/articles/s41588-023-01398-8/figures/1).

**Comment by joellembatchou** [comment]
Hi,

We'd recommend to first subset to the variants included in the genes used for the multi-gene tests before doing the merging to reduce the compute burden (e.g. getting the list of variants and passing it to `--extract` in PLINK with chr-specific files then merging the output files together). We have made a note of this feature request as a possible enhancement.

Cheers,
Joelle

**Comment by igumnov-daniel** [comment]
Does it mean Regenie doesn't take into account both chromosome and gene position in Set list file file? 
Thanks

**Comment by Wainberg** [comment]
@igumnov-daniel that's not the issue. Rather, it's that if you have one genotype file per chromosome, you can't run a burden test that involves variants from multiple chromosomes. This is because regenie has no way of reading in multiple files at once.

**Comment by Wainberg** [comment]
Yes, of course it's possible to work around it by merging across chromosomes, like I mentioned in the original post. It would be more convenient if this weren't necessary.
